### ISSUE TITLE
Es preferible desdoblarlo en mas de una prueba

### ISSUE THREAD

**Issue by evolentini** [franciscotimez/CESE_TSSE#1]
Para aplicar TDD es mejor dividirla en dos pruebas: primero una que revisar el estado de un led apagado y despues una que revisa el estado de un led encendido, con esas dos pruebas ya no factible devolver una constante.

https://github.com/franciscotimez/CESE_TSSE/blob/6216b3bfcb7f875ab7885f9653a1c6d511f21170/leds/test/test_leds.c#L69-L78

**Comment by franciscotimez** [comment]
No entiendo, acá yo estoy testeando la función Leds_State() que me devuelve el estado del led TRUE si el les esta encendido o FALSE si el led esta apagado.
O visto de otra forma, te referís a que es mejor apagar todos los leds y testear con TEST_ASSERT_FALSE(Led_State()) y en otra prueba encender todos los leds y testear con TEST_ASSERT_TRUE(Led_State()).

**Comment by evolentini** [comment]
Exacto, a que hagas dos pruebas simples separadas en lugar de una larga. No hace falta apagarlos porque ya arrancan apagados, y no hace falta prender todos si solo vas a testear un led encendido.

**Comment by franciscotimez** [comment]
https://github.com/franciscotimez/CESE_TSSE/blob/8d6b6fafd8e32c7867fd0d286dbdfbdee5676d60/leds/test/test_leds.c#L68-L77

Perfecto Esteban, lo acabo de corregir, espero tus comentarios!

**Comment by evolentini** [comment]
Excelente, tene la nota máxima en el TP3.

Saludos
